jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,654 @@
+    "\n",
+    "Once you've selected a Basemap, the Explorer updates to include that basemap in the view.\n",
+    "\n",
+    "Select the search icon and time cadence, for example monthly or quarterly. Then select one of the resulting basemap IDs, such as `global_monthly_2022_01_mosaic`. That's the value you'll be passing into the Orders API. \n",
+    "\n",
+    "### Get a basemap ID from the Basemap Viewer\n",
+    "\n",
+    "To pick a basemap, you can open the [Basemap Viewer](https://www.planet.com/basemaps/#/mosaic/) (you can learn more about the Basemap Viewer [here](https://developers.planet.com/docs/apps/basemapsviewer/)) and select a basemap from the left sidebar menu, using the filter to narrow down the basemap you want. \n",
+    "\n",
+    "Once you've selected a basemap, and selected the right arrow >, the basemap ID displays, such as `global_monthly_2022_01_mosaic`. That's the value you'll be passing into the Orders API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import Planet and Related Packages\n",
+    "\n",
+    "---\n",
+    "\n",
+    "Make sure you have Planet's Python package properly downloaded. You can find out more about this [here](https://developers.planet.com/docs/pythonclient/). Find your [API key](https://developers.planet.com/quickstart/apis/).\n",
+    "\n",
+    "Next set up a session by importing needed Python packages, pulling in your API Key, and make an initial request (to retrieve the Orders API parameters) to confirm a connection with the server."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth\n",
+    "import json\n",
+    "import time\n",
+    "import math"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"setup-session\"></a> \n",
+    "## Setup Session Using Orders API\n",
+    "---\n",
+    "The first step is to setup a session to access the Orders API. The Orders API can be used to download full mosaics, or to download individual parts of the mosaic known as quads. "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create Session to Access API\n",
+    "---\n",
+    "Use your Planet API key for successful authentication. Here, the API key is saved as an environment variable, which is imported into the notebook."
```

Comment:
Same comment as in the other notebook - add an explanation for GCP credentials, and mention it should be saved as an environment variable.